ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/CJavaDriverBasePlugin.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2017 Netflix, Inc.  All rights reserved.
+ */
+package com.netflix.ndbench.plugin.cass;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.ConsistencyLevel;
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.Session;
+import com.google.inject.Inject;
+import com.netflix.archaius.api.PropertyFactory;
+import com.netflix.ndbench.api.plugin.DataGenerator;
+import com.netflix.ndbench.api.plugin.NdBenchClient;
+import com.netflix.ndbench.api.plugin.common.NdBenchConstants;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author vchella
+ */
+public abstract class CJavaDriverBasePlugin implements NdBenchClient {
+
+    private static final org.slf4j.Logger Logger = LoggerFactory.getLogger(CJavaDriverBasePlugin.class);
+
+    protected static final String ResultOK = "Ok";
+    protected static final String ResutlFailed = "Failed";
+    protected static final String CacheMiss = null;
```

Comment:
Why do you need to declare `CacheMiss` as null. You could just use null in the two `return` statements in the code.